src/python/WMCore/Database/CMSCouch.py

```diff
@@ -353,11 +353,11 @@ def updateDocument(self, doc_id, design, update_func, fields=None, useBody=False
             updateUri = '/%s/_design/%s/_update/%s/%s?%s' % \
                         (self.name, design, update_func, doc_id, urllib.parse.urlencode(fields))
 
-            return self.put(uri=updateUri, decode=False)
+            return self.put(uri=updateUri, decode=decodeBytesToUnicode)
```

Comment:
```suggestion
            return self.put(uri=updateUri, decode=lambda x: decodeBytesToUnicodeConditional(x, condition=PY3))
```

I would only decode this in py3. It is true that no py2 unittest is failing, but I would change the behavior in py2 of our code as little as possible. what do you think?

(similar suggestion for other instances in this same file)

Reply:
by the way, thanks for providing this! i needed to backport this change to #10590 !

Reply:
Updating this code soon.